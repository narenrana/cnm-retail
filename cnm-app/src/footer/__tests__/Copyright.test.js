/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from "react";
import renderer from "react-test-renderer";
import Copyright from "../Copyright";

describe("Copyright test", () => {
  it("Copyright should match snapshot", () => {
    const component = renderer.create(<Copyright />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
