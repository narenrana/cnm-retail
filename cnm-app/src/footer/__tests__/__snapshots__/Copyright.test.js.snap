// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copyright test Copyright should match snapshot 1`] = `
<p
  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
>
  Copyright Â© 
   
  2020
  ALL RIGHT RESERVED
</p>
`;
